com.fdossena.flogo.yml

```diff
@@ -0,0 +1,57 @@
+app-id: com.fdossena.flogo
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+branch: stable
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+command: run.sh
+separate-locales: false
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --filesystem=host
+modules:
+  - name: flogo
+    buildsystem: simple
+    sources:
+      - type: file
+        only-arches:
+          - x86_64
+        url: https://downloads.fdossena.com/Projects/Flogo/flatpak/flogo-1.1.0-x86_64.zip
+        sha256: 1110b3eb8d90bb046ef771198484d47c178ecac58306cbb355c52b4e8460de77
+      - type: file
+        only-arches:
+          - aarch64
+        url: https://downloads.fdossena.com/Projects/Flogo/flatpak/flogo-1.1.0-aarch64.zip
+        sha256: 1b51f4feca95961a56b28626a76026f0bb6f30d88d72968d74f051ff331d08af
+      - type: file
+        path: run.sh
+      - type: file
+        url: https://downloads.fdossena.com/Projects/Flogo/flatpak/com.fdossena.flogo.desktop
```

Comment:
this and below: the URL isn't versioned. This mean it would break the build randomly when updating.

Why arent' they in the tarball instead?

Reply:
To be honest I just copied what the FreeTube yml was doing, it's my first time 😅
I'll add them to the zip files

Reply:
vibe coding isn't good.

Reply:
There's no need to be rude. I didn't vibe code anything, I'm just trying to figure out how flatpak works. I never used it before and it's quite complex.